lib/Runtime/Math/JavascriptMath.cpp

```diff
@@ -1095,6 +1095,12 @@ namespace Js
                     Output::Print(L"[PRNG:%x] INIT %I64x %I64x\n", scriptContext, seed0, seed1);
```

Comment:
Not related to your current change - but why this is not under any config trace?